Makefile

```diff
@@ -35,7 +35,7 @@ install-all-deps: install-build-deps install-site-deps ## Install all dependenci
 
 .PHONY: install-build-deps
 install-build-deps: ## Install dependencies (packages and tools)
-	install-build-deps.sh
+	./build/scripts/install-build-deps.sh
```

Comment:
This shouldn't be necessary due to setting `PATH` above (and isn't on Linux), this appears to be an issue with `make` and Mac OSX, can you look into that?